uradvd.c

```diff
@@ -634,6 +643,10 @@ static void parse_cmdline(int argc, char *argv[]) {
 	{
 		{"default-lifetime", required_argument, 0, 0},
 		{"rdnss", required_argument, 0, 1},
+		{"valid-lifetime", required_argument, 0, 2},
```

Comment:
With 4 more options, the magic numbers are getting a bit cumbersome. Let's introduce an enum instead:
```c
enum {
    OPT_DEFAULT_LIFETIME,
    OPT_RDNSS,
    ...
};
```
Then we can use that for long_options:
```c
    [OPT_DEFAULT_LIFETIME] = {"default-lifetime", required_argument, 0, 0},
    [OPT_RDNSS] = {"rdnss", required_argument, 0, 1},
    ...
```
and match on it in the switch.

Reply:
Maybe I'm missing a crucial part.
I get the first comment and it works like a charm.
bd2ecf13dc72f4c106287ecda142fb26ef03dc5e

But I can't figure out how your second comment will come together.
@neocturne How would your desired `getopt_long` call look like?

Reply:
I think I got it now, thanks :)
resolved in d94ea3279b0e6d9e7530200f83ab0c3766f21944